### ISSUE TITLE
Feature request: Ability to mofidy homeCoordinates after init

### ISSUE THREAD

**Issue by joglr** [torfsen/leaflet.zoomhome#3]
It would be nice to have a method for setting the home coordinates.

``` javascript
zoomHome.setHomeCoordinates([35.01267, 88.83459])
```

**Comment by answerquest** [comment]
Hi, this feature is very much needed. As an alternative to passing co-ordinates, how about passing bounds? Then I can pass in a layer's getBounds() whenever the layer has changes.

**Comment by answerquest** [comment]
@m1dst this is great! Request: can you add in a `setHomeBounds` method too? That would take as argument a `LatLngBounds` array, like from a `layer.getBounds()`.

Possible method to use : `getBoundsZoom` http://leafletjs.com/reference-1.3.0.html#map-getboundszoom

**Comment by m1dst** [comment]
@answerquest I've added the code and done a brief test.  Seems OK to me, give it a try and let me know if it doesn't do what you expected.

**Comment by answerquest** [comment]
@m1dst a [small correction](https://github.com/m1dst/leaflet.zoomhome/commit/c8d9ad6ef1a45175b7d452012912d8195494e68d) is needed, once that is done it works well! I added this code to the demo page to test:
```
var markerLayer = new L.geoJson(null);
var marker1 = L.marker([0,0]).addTo(markerLayer);
var marker2 = L.marker([20,20]).addTo(markerLayer);
markerLayer.addTo(map);
zoomHome.setHomeBounds( markerLayer.getBounds() );
```

**Comment by m1dst** [comment]
@answerquest Fixed in a subsequant update.  I had a variable called map which is why it worked for me.  Stupid typo.

**Comment by m1dst** [comment]
Thanks for all your feedback.  I have closed the original PR and moved the new code to a new branch targeting just this issue rather than pointing at my main branch.  I raised a new PR #12 where I have also minified the file.

**Comment by torfsen** [comment]
This has been closed via #12. Thanks, @m1dst!
